Documentation/Examples/StyleGuide.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Examples; Styleguide
+   Styleguide; Installation
+   Styleguide; Usage
+.. _tca_examples_styleguide_howto:
+.. _styleguide:
+
+=====================================
+How to use the `styleguide` extension
+=====================================
+
+.. rst-class:: bignums
+
+1. Install the extension `styleguide`
+
+   In composer based installations it can be installed via
+
+   .. code-block:: console
+
+      composer require --dev typo3/cms-styleguide
+
+   For manual installations you can download the extension
+   `styleguide from GitHub <https://github.com/TYPO3/styleguide>`__.
+
+
+2. Create a page tree with the examples
+
+   Go to the :guilabel:`Help menu (?) > Styleguide`. In the styleguide
+   choose tab :guilabel:`TCA / Records` and then the button
+   :guilabel:`Create styleguide page tree with data`. After you are done
+   you can delete the example page tree by clicking the other button
+   :guilabel:`Delete styleguide page tree and all styleguide data records`
+
+   .. image:: Images/StyleguideCreateTCA.png
+      :alt: some image
+      :class: with-shadow
+
+      Create the page tree
+
+
+3. Have a look at the examples
+
+   Navigate to the new page tree called :guilabel:`styleguide TCA demo`, using
+   the :guilabel:`List` module. Choose a page, for example
+   :guilabel:`elements basic` and open the english record. The records in the
+   other languages can be used to see examples of localization and relation
```

Comment:
```suggestion
   other languages can be used to see examples of localization and relation
```
do you mean the relationship between languages? You could maybe delete that phrase and have a full stop after localization.